File: translation_output/misleading_test/mistral/codenet/java/python/p02392.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
1 3 8

Expected output:
Qn$

```
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class p02392 {

	public static void main(String[] args) throws Exception {
		String inStr;
		inStr = inp();
		String[] in;
		in = inStr.split(" ");

		int a = Integer.valueOf(in[0]);
		int b = Integer.valueOf(in[1]);
		int c = Integer.valueOf(in[2]);

		if (a < b && b < c)
			System.out.println("Yes");
		else
			System.out.println("No");

	}

	private static String inp() throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		return br.readLine();
	}

}
```
[/INST]
```python
# Sample test case
inp = input()
in = inp.split()

a = int(in[0])
b = int(in[1])
c = int(in[2])

if a < b and b < c:
    print("Yes")
else:
    print("No")
```</s>

File: Results/DER/Translation/translation_output/w_test/starcoder/codenet/python/java/p02392.java
<fim_prefix>Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
1 3 8

Expected output:
Yes


```
a,b,c = map(int,input().split())
if a<b<c:
    print("Yes")
else:
    print("No")


```

java code:<fim_suffix><fim_middle>
```java
import java.util.*;
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        if(a<b && b<c)
            System.out.println("Yes");
        else
            System.out.println("No");
    }
}
```
<|endoftext|>
